README.md

```diff
@@ -25,6 +25,73 @@ The following features have been implemented:
 
  - Record `perf` data of the whole system, TiDB modules, or specific process. (disabled by default)
  - Check the file size of TiDB modules' data directories. (enabled by default)
+ - Collect and save log files of TiDB modules or system. (disabled by default)
+ - Collect ans save various config files of system. (disabled by default)
+ - Query PD API to collect runtime information of TiDB cluster. (enabled by default)
+
+### Usage
+
+    usage: insight.py [-h] [-o OUTPUT] [-p] [--pid PID] [--tidb-proc]
+                    [--perf-exec PERF_EXEC] [--perf-freq PERF_FREQ]
+                    [--perf-time PERF_TIME] [-l] [--syslog] [--config-file]
+                    [--pd-host PD_HOST] [--pd-port PD_PORT]
+
+    TiDB Insight Scripts
+
+    optional arguments:
+    -h, --help            show this help message and exit
```

Comment:
**S**how this help message and exit

Reply:
This line is generated by the `argparse` Python module by default.